data/core/style.lua

```diff
@@ -66,4 +66,9 @@ style.syntax["function"] = { common.color "#93DDFA" }
 style.syntax_fonts = {}
 -- style.syntax_fonts["comment"] = renderer.font.load(path_to_font, size_of_font, rendering_options)
 
+style.log = {
+  INFO  = { icon = "i", color = style.text },
+  ERROR = { icon = "!", color = { common.color "#ff6961" } }
```

Comment:
I don't think this is a good practice.
Why not use `style.accent`?

Reply:
It's because by default style accent is just a very bright white. It doesn't look obvious nor it represents error well.

Reply:
May be `style.nagbar`?
But we really do not have enough predefined colors for expressing ok/error, add/remove and the like.

Reply:
Now that we have control over the colors repo, we should add default colors for each theme for things like "info", "warning", "error", that even if we are not using directly, might be useful for plugins.

Reply:
I thought about the same. Although only two colors. https://github.com/lite-xl/lite-xl/pull/749